content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,50 @@ names in the `train` array below:
 
 ## Plot templates (data series only)
 
-Users have the ability to change the way data-series plots are displayed by
-modifying the [Vega-Lite specification](https://vega.github.io/vega-lite/), thus
-generating plots in the style that best fits the their needs. This keeps
-<abbr>DVC projects</abbr> programming language agnostic, as it's independent
-from user display configuration and visualization code.
+DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications to
+create visualizations out of users data. Pre-defined specifications are called
+templates.
 
-Built-in _plot templates_ are stored in the `.dvc/plots/` directory. The default
-one is called `default.json`. It can be changed with the `--template` (`-t`)
-option of `dvc plots show` and `dvc plots diff`. For templates in the
-`.dvc/plots/` directory, the path and the json extension are not required: you
-can specify only the base name e.g. `--template scatter`.
+DVC has few built in templates. They are stored within
+[`dvc-render`](https://github.com/iterative/dvc-render) library. The default one
```

Comment:
Optional suggestion: Can we drop the reference to dvc-render?

Reply:
I did remove it for now but do we plan to add a bunch? If so maybe document then in https://github.com/iterative/dvc-render#readme and just list a few here + link there.

Reply:
I think there should be a note somewhere about it - if we want people to contribute templates, we should include the link somewhere in `index`.

Reply:
> if we want people to contribute templates

If that's the intention then please do link to the repo somewhere.